README.adoc

```diff
@@ -331,9 +331,9 @@ Before you can use the filter class that you created, you need to create the `mi
 ----
 
 microprofile-config.properties
-[source, text, linenums, role="code_column"]
+[source, text, linenums, role='code_column tags=config']
 ----
-include::finish/src/main/webapp/META-INF/microprofile-config.properties[tags=config]
+include::finish/src/main/webapp/META-INF/microprofile-config.properties[]
 ----
 
 This configuration file is picked up automatically by MicroProfile Config and registers your filter
```

Comment:
Line 340, hotspot highlights the wrong line

Reply:
`mp.openapi.scan.disable = true` this shouldn't even be here since we tell the user to update the file to include this line later in the guide.

Reply:
Updated.

Reply:
> Line 340, hotspot highlights the wrong line

Updated.

Reply:
> mp.openapi.scan.disable = true
It's not there.